tox.ini

```diff
@@ -22,7 +22,9 @@ commands =
 
 [testenv:cover]
 commands =
-  python setup.py testr --coverage --testr-args='{posargs}'
+  coverage run -m testtools.run
```

Comment:
compared to other related PRs, we don't have "coverage erase" line here.....

Reply:
Yes, good point - will fix.